packages/app-defaults/package.json

```diff
@@ -50,15 +50,15 @@
     "@testing-library/jest-dom": "^6.0.0",
     "@testing-library/react": "^16.0.0",
     "@types/react": "^18.0.0",
-    "react": "^18.0.2",
-    "react-dom": "^18.0.2",
-    "react-router-dom": "^6.3.0"
+    "react": "^18.0.0",
+    "react-dom": "^18.0.0",
+    "react-router-dom": "^6.0.0"
```

Comment:
Kinda curious why we'd lower the ranges on these and the peer deps?

Reply:
Let's keep the most common minimum versions for now, so `18.0.2` for react and `6.3.0` for react-router, and use those across the entire project.